### ISSUE TITLE
Project members result empty when username_query is not supplied

### ISSUE THREAD

**Issue by willemo** [m4tthumphrey/php-gitlab-api#259]
When I try to get the project's members from the API, without supplying a `username_query` (I want to get all members), the result from the client is empty. When I call the URL (`https://gitlab.com/api/v4/projects/[projectID]/members`) manually, I get all members. When I do supply a `username_query` I do get the users that comply to that query.

So something seems to be not working correctly.

Code without query:

```php
$members = $client->projects()->members($projectId);
// $members is now an empty array
```

Code with query:

```php
$members = $client->projects()->members($projectId, 'willem');
// $members now contains one member: me
```

**Comment by m1guelpf** [comment]
@willemo Is getting all members using that endpoint ducumented anywhere?

**Comment by willemo** [comment]
@m1guelpf not that I know of. Is there another (documented) way of getting a project's members?

**Comment by m1guelpf** [comment]
@willemo The query parameter is marked as optional in the documentation...
[Somewhat related](https://gitlab.com/gitlab-org/gitlab-ce/issues/18672)?

**Comment by fbourigault** [comment]
https://github.com/m4tthumphrey/php-gitlab-api/blob/e37b78f2f0985cb2323adedc94d36a4f76ac2768/lib/Gitlab/Api/Projects.php#L241-L251 must be updated to use the `OptionsResolver` like in `Gitlab\Api\Projects::all()`.

**Comment by willemo** [comment]
@m1guelpf I don't think that issue is related.

Is there any other way for me to fetch the project's members?

**Comment by willemo** [comment]
Using the model-method (`$project->members()`) doesn't work either, sadly.

**Comment by fbourigault** [comment]
The `Gitlab\Api\Projects::getMembers` methods is buggy. When `$username_query` is not set, the `query` parameter must not be sent. That's why we used elsewhere the `OptionResolver` to keep parameters logic dry.
Feel free to send a PR. I would be happy to review it!

**Comment by tobiasetter** [comment]
this issue has already been fixed by #281 
@willemo please close the issue. thanks =)
